src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  transition: 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__description {
+    color: $main-text-color;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+
+    &:hover {
+      color: $main-text-color-hover;
+    }
+  }
+
+  &__code {
+    color: $secundary-text-color;
```

Comment:
There seems to be a typo in the variable name `$secundary-text-color`. It should be `$secondary-text-color` to follow standard spelling conventions.